DMCompiler/Optimizer/AnnotatedByteCodeWriter.cs

```diff
@@ -0,0 +1,421 @@
+using System;
+using System.Collections.Generic;
+using DMCompiler.Bytecode;
+using DMCompiler.Compiler;
+using DMCompiler.DM;
+
+namespace DMCompiler.Optimizer {
```

Comment:
File scoped namespaces pls I beg I am actually crying and sobbing IRL over this it's so bad 😭